samples/mixed/provider/src/provider_impl.rs

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+use log::{debug, info, warn};
+use proto::consumer::{consumer_client::ConsumerClient, RespondRequest};
+use proto::provider::{
+    provider_server::Provider, GetRequest, GetResponse, InvokeRequest, InvokeResponse, SetRequest,
+    SetResponse, SubscribeRequest, SubscribeResponse, UnsubscribeRequest, UnsubscribeResponse,
+};
+use std::collections::{HashMap, HashSet};
+use std::str::FromStr;
+use std::sync::{Arc, Mutex, MutexGuard};
+use tonic::{Request, Response, Status};
+
+use crate::vehicle::Vehicle;
+
+pub type SubscriptionMap = HashMap<String, HashSet<String>>;
+
+/// The ids for commands.
+const ACTIVATE_AIR_CONDITIOING: &str =
+    "dtmi:org:eclipse:sdv:vehicle:cabin:hvac:activate_air_conditioning;1";
+const SEND_NOTIFICATION: &str = "dtmi:org:eclipse:sdv:vehicle:cabin:hvac:send_notification;1";
```

Comment:
```
const SEND_NOTIFICATION: &str = 
    "dtmi:org:eclipse:sdv:vehicle:cabin:hvac:send_notification;1";

const SET_UI_MESSAGE: &str = 
   "dtmi:org:eclipse:sdv:vehicle:cabin:hvac:set_ui_message;1";
```

Reply:
The formatter (executed by running "cargo fmt") controls this.  It will wrap it if it considers it necessary.